v0/destinations/facebook_pixel/transform.js

```diff
@@ -527,6 +527,25 @@ const responseBuilderSimple = (message, category, destination) => {
     }
   }
 
+  // content_category should only be a string ref: https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/custom-data
+
+  if (
+    customData &&
+    customData.content_category &&
+    typeof customData.content_category !== "string"
+  ) {
+    if (Array.isArray(customData.content_category)) {
```

Comment:
The array can be empty.

Reply:
In that case empty string `""` will be returned, which is supported by facebook. And this is already being done in `handleProduct` function, line no : 199

Reply:
Cool.